examples/test.cpp

```diff
@@ -1,6 +1,33 @@
-#include "../include/featurelib.h"
-
+#include "../include/FeatureLib.h"
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui.hpp>
+std::vector<uint8_t> matToVector(const cv::Mat &mat) {
+  std::vector<uint8_t> result;
+  cv::Mat cont = mat.isContinuous() ? mat : mat.clone();  // make sure the Mat is continuous
+  cont.reshape(1, 1).copyTo(result);  // copy from a 1D Mat
+  return result;
+}
 int main() {
```

Comment:
Add an empty newline between the functions

Reply:
done